apps/console/src/features/organizations/__mocks__/organization.ts

```diff
@@ -16,6 +16,7 @@
 * under the License.
 */
 import { OrganizationManagementConstants } from "../constants";
+import { OrganizationListInterface } from "../models";
```

Comment:
It will be better if we can add a test to check this enable/disable feature as well.

Reply:
Done